ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRao.java

```diff
@@ -46,9 +53,37 @@ public String getVersion() {
         }
         crac.generateValidityReport(network);
 
+        // compute maximum loop flow value F_(0,all)_MAX, and update it for each Cnec in Crac
+        LoopFlowExtensionParameters loopFlowExtensionParameters = parameters.getExtension(LoopFlowExtensionParameters.class);
+        CracLoopFlowExtension cracLoopFlowExtension = crac.getExtension(CracLoopFlowExtension.class);
+        if (!Objects.isNull(loopFlowExtensionParameters) && useLoopFlowExtension(loopFlowExtensionParameters)
+            && !Objects.isNull(cracLoopFlowExtension)) {
+            //For the initial Network, compute the F_(0,all)_init
+            LoopFlowComputation loopFlowComputation = new LoopFlowComputation(network, crac, cracLoopFlowExtension.getGlskProvider(), cracLoopFlowExtension.getCountriesForLoopFlow());
```

Comment:
can be renamed to initialLoopFlowComputation

Reply:
OK